api/capi/include/nntrainer.h

```diff
@@ -436,12 +449,13 @@ int ml_train_dataset_add_generator(ml_train_dataset_h dataset,
                                    ml_train_datagen_cb cb, void *user_data);
 
 /**
- * @brief Add data file to @a dataset
+ * @brief Adds data file to @a dataset.
  * @details Use this function to add a data file from where data is retrieved.
- * @remarks %http://tizen.org/privilege/mediastorage is needed if @a dataset is
- * saved to media storage.
- * @remarks %http://tizen.org/privilege/externalstorage is needed if @a model is
- * saved to external storage.
+ * @since_tizen 6.5
+ * @previliege %http://tizen.org/privilege/mediastorage is needed if @a dataset
```

Comment:
Simply put privilege name after '@privilege' tag: '@privilege %http://tizen.org/privilege/externalstorage' Note that currently tag name has typo ('@previliege' -> '@privilege')
The same applies to privileges in `ml_train_model_construct` function.

Reply:
the function does not always require privilege but for some cases it needs, do we put privilege tag in this case too?

Reply:
I think `privilege` tag should be added and cases when it is necessary should be described in `details`.

Reply:
Done :)